src/Repository/FunderRepository.php

```diff
@@ -39,6 +39,19 @@ public function remove(Funder $entity, bool $flush = false): void
         }
     }
 
+    /**
+     * @return Funder[] Returns an array of Funder objects
```

Comment:
Add title and parameter to this. `@return` is optional. Also, not correct, just returns an array.